src/rng.rs

```diff
@@ -325,6 +325,31 @@ pub trait Rng: RngCore {
     {
         self.random()
     }
+
+    /// Alias for [`Rng::random_range`].
+    #[inline]
+    #[deprecated(since = "0.9.0", note = "Renamed to `random_range`")]
```

Comment:
Maybe it's better to do a deprecating backport and release in the v0.8 branch and remove these methods completely in v0.9?

Reply:
I don't think we should add a `#[deprecated]` notice in a patch release because (a) it may break some builds (https://doc.rust-lang.org/cargo/reference/semver.html#new-lints) and (b) it effectively forces users who don't want to see a warning to update their code just because they updated to the next patch version.

Since the next release is a breaking release, we could just *not* add these `#[deprecated]` fn wrappers, but they potentially ease migration and I don't see much harm in leaving them in until 1.0 (or 0.10).